Firmware/ultralcd.cpp

```diff
@@ -5960,28 +5409,25 @@ uint8_t choose_menu_P(const char *header, const char *item, const char *last_ite
 }
 
 char reset_menu() {
-    const uint8_t items_no =
-#ifdef SNMM
-        6;
-#else
-        5;
-#endif
-    static int8_t first = 0;
-    int8_t enc_dif = 0;
+	const uint8_t items_no = 5;
```

Comment:
maybe we can move this after item[] and use ``sizeof(item)/sizeof(*item)``

Reply:
David said not to use *item in array size snippet and use item[0] instead. I'm updating that now

Reply:
Should be fully equivalent semantically, right?
Either way is fine for me.

Reply:
Yes, it is equivalent.

Reply:
Pushed an update to this.